sympy/core/tests/test_evalf.py

```diff
@@ -535,3 +535,6 @@ def test_issue_14601():
     e2 = e.evalf(subs=subst)
     assert float(e2) == 0.0
     assert float((x + x*(x**2 + x)).evalf(subs={x: 0.0})) == 0.0
+def test_issue_11151():
+    e=Sum(0,(x,1,2))
+    assert n.evalf(e,15,{}) == n.evalf(S(0),15,{})
```

Comment:
How is `n` defined? Is is just a Symbol?

Reply:
The example given in [this comment](https://github.com/sympy/sympy/issues/11151#issuecomment-473603000) would probably make a good test with `assert simplify(expr4) == 0`.

Reply:
n is a symbol . But I changed to evalf. shall I change the test?